data/mods/Aftershock/items/obsolete.json

```diff
@@ -17,6 +17,17 @@
     "looks_like": "flour",
     "fun": -5
   },
+  {
+    "type": "mutation",
```

Comment:
This isn't an obsolete *item*. 😛

Reply:
In other words, I think you want this at https://github.com/CleverRaven/Cataclysm-DDA/blob/71c6ab6f2f00cac9b86d025defcbe077ea2930d4/data/mods/Aftershock/player/obsolete.json